app/build.gradle

```diff
@@ -12,6 +11,9 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        jackOptions {
```

Comment:
can anyone fill me in on what this is? 

it's fine I just want to know why it's there (don't wanna have surprises later :P)

Reply:
I am also unfamiliar with who added that and would also like to know what it is/does

Reply:
This was to support compiling with Java 8. I just did some research on it and it's sorta a hack, so I'm reverting this change. For now, we'll just use Java 7

Reply:
ya no worries. glad we know what's up. 

i think there was a problem with using java 8 somewhere, I just can't remember what it was.